docs/Configuring.md

```diff
@@ -316,13 +316,13 @@ Cromwell also accepts [Workflow Options](wf_options/Overview#call-caching-option
 
 When running a job on the Config (Shared Filesystem) backend, Cromwell provides some additional options in the backend's config section:
 
-```
+```HOCON
       config {
-        ...
+        #...
```

Comment:
I actually think removing these `...`s would be fine given that this is HOCON and is extending an existing config file anyway